scripts/install/install_macos.sh

```diff
@@ -566,22 +571,38 @@ install_package()
   decrease_indent
 }
 
-# create_manager_yml creates the manager.yml at the specified path, containing opamp information.
-create_manager_yml()
+create_supervisor_config()
 {
-  manager_yml_path="$1"
-  if [ ! -f "$manager_yml_path" ]; then
-    info "Creating manager yaml..."
-
-    # Note here: We create the file and change permissions of the file here BEFORE writing info to it
-    # We do this because the file may contain a secret key, so we want 0 window when the
-    # file is readable by anyone other than root
-    command printf '' >> "$manager_yml_path"
-    chmod 0600 "$manager_yml_path"
-
-    command printf 'endpoint: "%s"\n' "$OPAMP_ENDPOINT" > "$manager_yml_path"
-    [ -n "$OPAMP_LABELS" ] && command printf 'labels: "%s"\n' "$OPAMP_LABELS" >> "$manager_yml_path"
-    [ -n "$OPAMP_SECRET_KEY" ] && command printf 'secret_key: "%s"\n' "$OPAMP_SECRET_KEY" >> "$manager_yml_path"
+  supervisor_yml_path="$1"
+  if [ ! -f "$supervisor_yml_path" ]; then
+    info "Creating supervisor config..."
+
+        # Note here: We create the file and change permissions of the file here BEFORE writing info to it.
+    # We do this because the file contains the secret key.
+    # We do not want the file readable by anyone other than root/obseriq-otel-collector.
+    command printf '' >> "$supervisor_yml_path"
+    chown observiq-otel-collector:observiq-otel-collector "$supervisor_yml_path"
```

Comment:
I get this: 
`chown: observiq-otel-collector: illegal group name`

If we want this to have a separate group/user, we'll need to create the group/user first.
Also, if we're gonna do this, we should make everything in this directory belong to this user/group (chown -R)

We can just remove this line for now and come back to it later too, we've had everything belonging to root here for quite a while.

Reply:
That works for me. This was something I pulled over from the linux script, but that has the luxury of a pre-install script that sets/creates the group. I'll take it out